frontend/src/screens/WelcomeScreen.js

```diff
@@ -8,12 +8,11 @@ import TextField from "@material-ui/core/TextField";
 const WelcomeScreen = () => {
   const context = useContext(MyContext);
   console.log(context.state.phase);
-  const [roomToJoin, setRoom] = useState("");
+  const [roomIDToJoin, setRoom] = useState("");
 
-  const onChange = (event) => {
-    const value = event.target.value;
-    setRoom(value);
-    console.log(roomToJoin);
+  const onRoomInputChange = (event) => {
+    setRoom(event.target.value);
```

Comment:
what is setRoom

Reply:
its a method we got from useState hook before, its fine. we edit the room before we send it